### PR TITLE
Add/Upgrade task container types on node onfleet types

### PR DESCRIPTION
**Pull request by JMalaval**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://docs.onfleet.com/reference#update-task

Even if it is not explicit because there is nothing written in the updateTask or createTask documentation. In the example used, there is the update of a container with the type `TEAM`

```
{
   "notes":"Patxi\\'s deep dish meat lover special, 2 bottles of Basque cider",
   "container":{
      "type":"TEAM",
      "team":"fwflFNVvrK~4t0m5hKFIxBUl"
   }
}
```

### CODE REVIEW

File: types/onfleet__node-onfleet/Resources/Tasks.d.ts [hunk 1]
```diff
@@ -164,6 +163,21 @@ declare namespace Task {
     eta: number;
     trackingViewed: boolean;
   }
+
+  interface WorkerTaskContainer {
+    type: 'WORKER';
+    worker: string;
+  }
```
Comment (marcobeltempo) [inline types/onfleet__node-onfleet/Resources/Tasks.d.ts:170]: nit: let's add a line break after the container interfaces 
```suggestion
  }

```

File: types/onfleet__node-onfleet/Resources/Tasks.d.ts [hunk 2]
```diff
@@ -164,6 +163,21 @@ declare namespace Task {
...
+  interface OrganizationTaskContainer {
+    type: 'ORGANIZATION';
+    organization: string;
+  }
```
Comment (marcobeltempo) [inline types/onfleet__node-onfleet/Resources/Tasks.d.ts:174]:
```suggestion
  }

```

**Review by marcobeltempo — CHANGES_REQUESTED** [review]
Thanks for the containers updates @JMalaval. Just had some formatting comments but other than that the changes look good 👍🏼

**Review by marcobeltempo — APPROVED** [review]

File: types/onfleet__node-onfleet/Resources/Tasks.d.ts [hunk 3]
```diff
@@ -164,6 +163,23 @@ declare namespace Task {
...
+  interface WorkerTaskContainer {
+    type: 'WORKER';
+    worker: string;
+  }
+
+  interface OrganizationTaskContainer {
+    type: 'ORGANIZATION';
+    organization: string;
+  }
+
```
Comment (marcobeltempo) [inline types/onfleet__node-onfleet/Resources/Tasks.d.ts:176]: @JMalaval the linter is failing due to a trailing whitespace
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50771/checks#step:5:106
```suggestion
 
```

**Review by marcobeltempo — CHANGES_REQUESTED** [review]

**Comment by JMalaval** [comment]
Sorry for all those stupids mistake 😞

**Review by marcobeltempo — APPROVED** [review]
Thanks for the update @JMalaval

**Comment by marcobeltempo** [comment]
Ready to merge

**Pull request merged**